### PR TITLE
fix(staking-ui): staking stats mobile component

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: libs/staking/ui-kit/src/components/staking-info/staking-info.tsx [hunk 1]
```diff
@@ -97,3 +106,75 @@ export function RewardsInfo({
...
+        className={clsx(
+          'flex gap-x-[4px] text-[13px] font-[500] leading-[20px]',
+          isGreen ? 'text-[#01B26E]' : 'text-white',
+        )}
+      >
+        {value}
+        <span>{symbol.toUpperCase()}</span>
+      </div>
+    </div>
+  );
```
Comment (olegshilov) [inline libs/staking/ui-kit/src/components/staking-info/staking-info.tsx]:
```suggestion
    <div className="flex flex-col items-start gap-[6px]">
      <div>
        <div className="text-[12px] font-[600] uppercase leading-[1.2em] text-white/50">
          {title}
        </div>
      </div>
      <div>
        <div
          className={clsx(
            'text-[13px] font-[500] leading-[20px]',
            isGreen ? 'text-[#01B26E]' : 'text-white',
          )}
        >
          {value}&nbsp;{symbol.toUpperCase()}
        </div>
      </div>
    </div>
  );
```

File: libs/staking/ui-kit/src/components/staking-info/staking-info.tsx [hunk 2]
```diff
@@ -97,3 +106,75 @@ export function RewardsInfo({
...
+  return (
+    <div className="flex items-start gap-[16px] overflow-x-auto px-[16px] py-[20px]">
+      <div>
+        <WalletIcon />
+      </div>
+      <StakingInfoAmountBlockMobile
+        title="Available"
+        value={balance}
+        symbol={symbol}
+      />
```
Comment (olegshilov) [inline libs/staking/ui-kit/src/components/staking-info/staking-info.tsx]:
```suggestion
  <div className="flex-initial">
    <StakingInfoAmountBlockMobile
      title="Available"
      value={balance}
      symbol={symbol}
    />
  </div>
```

File: libs/staking/ui-kit/src/components/staking-info/staking-info.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/staking/ui-kit/src/components/staking-info/staking-info.tsx]: Done

File: libs/staking/ui-kit/src/components/staking-info/staking-info.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/staking/ui-kit/src/components/staking-info/staking-info.tsx]: Done

**Review by olegshilov — APPROVED** [review]

**Pull request merged**